### ISSUE TITLE
[next-build] Create CODEOWNERS file

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#19647]
## Description
The `next-build` repository does not have a CODEOWNERS file yet. We should create it mirroring the general structure of the [CODEOWNERS file in content-build](https://github.com/department-of-veterans-affairs/content-build/blob/main/.github/CODEOWNERS) as the ownership will likely be the same.

Slack thread for context: https://dsva.slack.com/archives/C07B86KV4P4/p1730236643814469

## Acceptance criteria
- [ ] CODEOWNERS file is created
- [ ] @timcosgrove has reviewed and approved the PR
- [ ] At least one member of the Sitewide team has reviewed and approved the PR
- [ ] Platform has reviewed and approved the PR
- [ ] CODEOWNERS PR is merged into `next-build`

**Comment by randimays** [comment]
I thought it would make the most sense for Sitewide to take the first pass at it and then collaborate with Tim to refine, given how busy he is. But I'm open to passing this to him if he'd prefer it that way (or if we can't prioritize it). I thought it should get worked soon since IIR is going to start contributing and we need to nail down who should be looking at the code (formally).

**Comment by FranECross** [comment]
Thanks, @randimays , and I agree with your thoughts. @jilladams do we need to check with Tim before our folks take a run at this?

**Comment by jilladams** [comment]
We've done so here: https://dsva.slack.com/archives/C07B86KV4P4/p1730237424480579

I think they'd be grateful to have us get something into PR for review.

**Comment by randimays** [comment]
I dropped the CODEOWNERS PR for review here: https://dsva.slack.com/archives/C01SR56755H/p1734032435334349. It spurred quite the discussion, so this ticket may sit for a while as things get worked out behind the scenes.

**Comment by randimays** [comment]
Michelle confirmed that discussion for this will wait until after the holidays. @FranECross @jilladams not sure if we want to leave this ticket in its current state.

**Comment by jilladams** [comment]
Yep - I've been reporting it as blocked, so I think it's ok to sit here til we get more news. If you'd prefer to unassign yourself to reduce noise for now, that's fine.

**Comment by jilladams** [comment]
This came up on office hours today and Tim clarified: there's platform level work here to determine: 
1. Code structure changes that need to be made in order to enable Sitewide and other contributing teams to be able to modify the necessary files to get work done. 
2. Thoughtful determination of which Next files should / shouldn't be owned by the core platform team vs. contributing teams. 

So: though Sitewide took this on as a nice thing to start out with, with the idea / discussion that we start from the content-build codeowners file, that's not really gonna work. So this ticket needs to go back to AP to own. 

Reassigning. FYI @mmiddaugh @gracekretschmer-metrostar @EWashb . Feel free to come back and modify the short term notes / change here as needed once we formally revisit the topic next year.
